webpack/ForemanInventoryUpload/Components/InventorySettings/InventorySettingsActions.js

```diff
@@ -0,0 +1,32 @@
+import { get, post } from 'foremanReact/redux/API';
+import { addToast } from 'foremanReact/redux/actions/toasts';
+import {
+  INVENTORY_SETTINGS,
+  INVENTORY_GET_SETTINGS_PATH,
+  INVENTORY_SET_SETTINGS_PATH,
+} from './InventorySettingsConstants';
+
+export const getSettings = () =>
+  get({
+    key: INVENTORY_SETTINGS,
+    url: INVENTORY_GET_SETTINGS_PATH,
+  });
+
+export const setSetting = (setting, request) => dispatch => {
+  dispatch(
+    post({
+      key: INVENTORY_SETTINGS,
+      url: INVENTORY_SET_SETTINGS_PATH,
+      params: setting,
+      handleError: error =>
+        dispatch(
+          addToast({
+            sticky: true,
+            type: 'error',
+            message: error.message,
+          })
```

Comment:
I guess if it won't be a sticky one :)
```suggestion
      errorToast: error => error.message
```